### PR TITLE
Updated README.md with punctuation

### PR DESCRIPTION
**Pull request by baishakhi9931**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,34 +26,34 @@ This file should be placed in the root of the repository and set to your liking.
...
 - The default behavior is to receive Imgbot PRs as images require optimization
 
 **ignoredFiles**
 
-- optional
+- Optional
 - Accepts the syntax for searchPattern on [Directory.EnumerateFiles()](https://docs.microsoft.com/en-us/dotnet/api/system.io.directory.enumeratefiles)
 - Limits the images optimized by Imgbot by essentially ignoring them
-- When ignoring by filename no path is necessary, when ignoring by foldername full path from root is necessary
+- When ignoring by filename no path is necessary, when ignoring by folder name full path from root is necessary
```
Comment (dlemstra) [inline README.md:45]: We should probably change `filename` to `file name` also.

**Review by dlemstra — COMMENTED** [review]
Thanks for your pull request. Could you also add the other change that I suggested?

**Comment by Prakhar998** [comment]
make space between searchPattern to search Pattern
Accepts the syntax for searchPattern on Directory.EnumerateFiles()
Limits the images optimized by Imgbot by essentially ignoring them

**Comment by dlemstra** [comment]
Thanks for your additional changes @baishakhi9931.

**Pull request merged**